starcounter-include.html

```diff
@@ -315,6 +315,16 @@
             return range.createContextualFragment(htmlStr);
         }
 
+        function findCompositionProvider(obj) {
```

Comment:
Single call `findCompositionProvider(HTMLElement)` costs about 0.02ms, so I don't think there is a big overhead.

We can add some memoization to the function:

```js
        var memoizedKey;
        function findCompositionProvider(obj) {
            if (memoizedKey) {
                return obj[memoizedKey];
            }
            for (var i in obj) {
                if (obj.hasOwnProperty(i) && obj[i] != null && obj[i].hasOwnProperty("PartialId")) {
                    memoizedKey = i;
                    return obj[i];
                }
            }
        };
```